database/migrations/2019_11_20_155351_add_objectguid_column.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddObjectguidColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('agents', function (Blueprint $table) {
+            $table->string('objectguid')->unique()->nullable()->after('id');
```

Comment:
What's the purpose of `objectguid`? Just so I know for myself.

Reply:
The objectguid is a new column required in the adldap2-laravel package upgrade from 5.* to 6.* when using the DatabaseUserProvider: https://adldap2.github.io/Adldap2-Laravel/#/upgrading?id=authentication. It's supposed to "allow usernames to change in your directory and synchronize properly in your database".

Reply:
Cool, I did experience an issue when trying to run the migration locally:

```
In StreamHandler.php line 98:
                                                                               
  Missing stream url, the stream can not be opened. This may be caused by a premature call to close().
```

It did run other migrations though, so I don't think it's my mysql connection. Have you have any issues?